app/templates/photo-albums/photo-album/photos/index.hbs

```diff
@@ -28,3 +28,45 @@
     </div>
   {{/each}}
 </div>
+
+{{#if modalIsOpen}}
+  {{#ember-wormhole to="ember-wormhole-destination"}}
+    <div class="modal d-block">
```

Comment:
Gebruik liever ember-bootstraps Modal. Dat doet als het goed is de meeste dingen voor je, inclusief classes (en zou upgraden makkelijker moeten maken)

Reply:
Ja, probleem daarmee is dat die bootstrap dingen naar mijn idee niet werken, dat is ook waarom de exif data dropdown niet werkt volgens mij. Ik heb juist gekozen voor de ember oplossing. Waarom zou ik beter de ember-bootstrap modal moeten gebruiken?

Reply:
Er zijn verschillende redenen om ember-bootstrap te gebruiken in plaats van zelf de classes uit te zoeken:
1) Je hoeft niet de Bootstrap documentatie te lezen om een modal te kunnen schrijven (maar wel de ember-bootstrap docs)
2) Je vergeet minder snel classes
3) Ik vind het er leesbaarder (explicieter) uit zien
4) Upgraden naar een nieuwe minor versie van Bootstrap is in theorie makkelijker omdat iemand anders al heeft gecheckt wat er precies anders wordt
5) Je hoeft niet zelf de ember-wormhole te doen
6) De Toggle event handler krijg je er gratis bij

De reden dat de exif-data niet werkte was overigens omdat ik zelf ook niet ember-bootstrap gebruikte, iets dat ik achteraf gezien wel had willen doen :grin:

Reply:
Een laatste reden die alleen van toepassing is bij formulieren: ember-bootstrap maakt validaties een flink stuk makkelijker, mits onze formulieren wat meer 'vanilla' worden

Reply:
Check, ik zal er naar kijken